### ISSUE TITLE
Sidebar: Active Tab is not Indicated

### ISSUE THREAD

**Issue by auromita** [UMD-Digital/umd_terp#8]
#### Sidebar: Screenreader is reading sidebar contents but not saying which is the active tab on the list. 

Users should know which section is being focused on. 

###### Screenshot Below: 

![image](https://user-images.githubusercontent.com/67974395/88322944-73c8a200-ccef-11ea-9d96-d210ab15ccea.png)

**Comment by drogers98** [comment]
@auromita Currently, active link in the sidebar menu is being displayed with:

`aria-current="page"`

This is in accordance with best practice as shown here: https://www.w3.org/WAI/tutorials/menus/structure/#using-wai-aria

If there is something else additional desired, or this needs to be changed, just let us know what that is.

**Comment by magnessjo** [comment]
Hi @drogers98, I agree `aria-current="page"` should be providing an indication for user.  Jaws or other screenreader might pick up on differently.  My suggestion would be to add in a screenreader only span so a user will get more context for the element.  I would do something like `<span class="sr-only">The Current Page is</span> Basic Pages`.  I am assuming that is straightforward since we are already adding `aria-current` dynamically?  

Let me know if you disagree and we can discuss on the next call.

**Comment by drogers98** [comment]
Fixed in 8.1.26
